DcNotificationService/NotificationService.swift

```diff
@@ -6,9 +6,12 @@ class NotificationService: UNNotificationServiceExtension {
 
     override func didReceive(_ request: UNNotificationRequest, withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void) {
         guard let bestAttemptContent = (request.content.mutableCopy() as? UNMutableNotificationContent) else { return }
+        let nowTimestamp = Double(Date().timeIntervalSince1970)
+        UserDefaults.pushToDebugArray("🤜")
 
-        if UserDefaults.mainAppRunning {
-            contentHandler(silenceNotification(bestAttemptContent))
+        if UserDefaults.mainIoRunning {
+            UserDefaults.pushToDebugArray("ABORT4")
```

Comment:
it is just a short, unique, greppable string, useful only for iOS devs in combination with the code to see where things exit/flow.

it is short to recognise patterns, this is sth. missing in a usual log (also, we currently do not have the log at "view log ... and also not sure if it would work cross-process). this is how patterns currently look like:

<img width=320 src=https://github.com/deltachat/deltachat-ios/assets/9800740/bae602c3-9f6b-4a52-b434-0fc60eb8300d>